tests/testCameraSys.py

```diff
@@ -38,47 +38,39 @@ def testBasics(self):
         for sysName in ("pupil", "pixels"):
             for detectorName in ("", "det1", "det2"):
                 cameraSys = cameraGeom.CameraSys(sysName, detectorName)
-                self.assertEquals(cameraSys.getSysName(), sysName)
-                self.assertEquals(cameraSys.getDetectorName(), detectorName)
-                self.assertEquals(cameraSys.hasDetectorName(), bool(detectorName))
+                self.assertEqual(cameraSys.getSysName(), sysName)
+                self.assertEqual(cameraSys.getDetectorName(), detectorName)
+                self.assertEqual(cameraSys.hasDetectorName(), bool(detectorName))
 
                 noDetSys = cameraGeom.CameraSys(sysName)
-                self.assertEquals(noDetSys.getSysName(), sysName)
-                self.assertEquals(noDetSys.getDetectorName(), "")
+                self.assertEqual(noDetSys.getSysName(), sysName)
+                self.assertEqual(noDetSys.getDetectorName(), "")
                 self.assertFalse(noDetSys.hasDetectorName())
 
                 camSysPrefix = cameraGeom.CameraSysPrefix(sysName)
-                self.assertEquals(camSysPrefix.getSysName(), sysName)
+                self.assertEqual(camSysPrefix.getSysName(), sysName)
 
                 if detectorName:
-                    self.assertFalse(cameraSys == noDetSys)
-                    self.assertTrue(cameraSys != noDetSys)
+                    self.assertNotEqual(cameraSys, noDetSys)
                 else:
-                    self.assertTrue(cameraSys == noDetSys)
-                    self.assertFalse(cameraSys != noDetSys)
+                    self.assertEqual(cameraSys, noDetSys)
 
-                self.assertTrue(cameraSys != camSysPrefix)
-                self.assertTrue(noDetSys != camSysPrefix)
-                self.assertFalse(cameraSys == camSysPrefix)
-                self.assertFalse(noDetSys == camSysPrefix)
+                self.assertNotEqual(cameraSys, camSysPrefix)
+                self.assertNotEqual(noDetSys, camSysPrefix)
```

Comment:
Please restore `assertNotEqual(foo != bar)`, preferably with a comment such as `# use this instead of assertNotEqual to test operator!=`.

You can probably retain `assertEqual(foo, bar)` instead of `assertTrue(foo == bar) as that is surely a safe substitution.